index.js

```diff
@@ -7,33 +7,48 @@ let util = require('util');
 let events = require('events');
 let xmlResponseParser = require('parsexmlresponse');
 
-let httpServerPort;
+var httpServerEmitter = new events();
+var httpServerStarting = false;
+var httpServerStarted = false;
+var httpServerPort;
```

Comment:
I can certainly do that. I think `let` was causing problems with Node.js v6, I'll test and let you know. The issues I was encountering were probably due to the pre-async design of the http server initialization.